test_conformance/printf/test_printf.cpp

```diff
@@ -946,9 +924,10 @@ int main(int argc, const char* argv[])
         }
     }
 
-    if (getTempFileName() == -1)
+    strcpy(gFileName, get_temp_filename());
+    if (strlen(gFileName) == 0)
     {
-        log_error("getTempFileName failed\n");
+        log_error("get_temp_filename failed\n");
         return -1;
     }
```

Comment:
would be good to release the temp file name and avoid the leak here as well.

Reply:
That's correct. I think I didn't push previous commit.